content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+---
+title: Istio Deployment Models
+description: Describes the system models that impact your overall Istio depolyment.
+weight: 60
+keywords:
+- single-cluster
+- multiple-clusters
+- control-plane
+- tenancy
+- networks
+- identity
+- trust
+- single-mesh
+- multiple-meshes
+aliases:
+- /docs/concepts/multicluster-deployments/
+---
+
+Important system models impact your overall Istio deployment model. This page
+discusses the options for each of these models and describes how you can
+configure Istio to address them.
+
+## Cluster models
+
+A {{< gloss >}}cluster{{< /gloss >}} is a group of compute nodes within physical
+proximity that, typically, can reach each other directly. For isolation,
+performance, and high availability, most cloud providers confine clusters to
+availability zones and regions. Production systems, depending on their
+requirements, can run across multiple clusters spanning a number of zones or
+regions, leveraging cloud load balancers to handle things like locality and
+zonal or regional failover.
+
+In most cases, clusters represent boundaries for configuration and endpoint
+discovery. For example, each Kubernetes cluster has an API Server which manages
+the configuration for the cluster as well as serving
+{{< gloss >}}service endpoint{{< /gloss >}} information as pods are brought up
+or down. Since Kubernetes configures this behavior on a per-cluster basis, this
+approach helps limit the potential problems caused by incorrect configurations.
+
+In Istio, you can configure a single service mesh to span any number of
+clusters.
+
+### Single cluster
+
+In the simplest case, you can confine an Istio mesh to a single cluster. A
+single cluster usually operates over a [single network](#single-network), but it
+varies between infrastructure providers. In single cluster and single network
+models, you commonly deploy a single Istio control plane, which results in the
+simplest Istio deployment.
+
+{{< image width="50%"
+    link="./single-cluster.svg"
+    alt="A service mesh with a single cluster"
+    title="Single cluster"
+    caption="A service mesh with a single cluster"
+    >}}
+
+Single cluster deployments offer simplicity, but lack fault isolation and
+failover. If you need higher availability, you should use multiple clusters.
+
+### Multiple clusters
+
+You can configure your mesh to include multiple clusters; such configurations
+are known as **multi-cluster**. Using multiple clusters
+affords the following capabilities beyond that of a single cluster model:
+
+- Fault isolation and failover: `cluster-1` goes down, failover to `cluster-2`.
+- Location-aware routing and failover: Send requests to the nearest service.
+- Configuration isolation: A wrong configuration only affects a single cluster.
```

Comment:
> A wrong configuration only affects a single cluster.

Is it correct in the case of single/shared control plane? A wrong Istio configuration can affect multiple clusters.